code/obj/item/mob_parts/robot_parts.dm

```diff
@@ -259,6 +259,8 @@ ABSTRACT_TYPE(/obj/item/parts/robot_parts/head)
 				if (M.amount < 1)
 					user.drop_item()
 					qdel(M)
+				else
+					M.inventory_counter.update_number(M.amount)
```

Comment:
I think it'd be nicer for all of these to use `change_stack_amount`, which has everything covered wrt the sheets.